docs/appendix.rst

```diff
@@ -137,6 +137,20 @@ This documentation uses a few special terms to refer to Python types:
       No tilde expansion is done, but ``*``, ``?``, and character ranges
       expressed with ``[]`` will be correctly matched.
 
+   traditional operations
+      The CIM-XML operations originally defined by the DMTF in :term:`CIM-XML`
+      operations defined in the DMTF specification CIM Operations Over HTTP
+      (:term:`DSP0200`) for requesting multiple instances including
```

Comment:
"CIM-XML operations" used twice, and the reference to DSP0200 could be more compact.
How about: "The CIM-XML operations originally defined by the DMTF in :term:`DSP0200` for ..."

Reply:
Done